src/sentry/integrations/slack/integration.py

```diff
@@ -133,6 +135,19 @@ def get_pipeline_views(self) -> Sequence[View]:
 
     def _get_team_info(self, access_token: str) -> Any:
         # Manually add authorization since this method is part of slack installation
+
+        # first try with new SDK client (not attached to integration)
+        try:
+            client = WebClient(token=access_token)
```

Comment:
i went with this because the new `SlackSdkClient` subclasses `WebClient`, and uses `integration_id` to fetch the access token to initialize the parent. we don't have integration_id here because it doesn't exist yet.

if we already have the access token we don't need to use the SlackSdkClient unless we care about tracking the metric via DD. we also don't need to track successes and fails in order to disable the integration here because it doesn't exist